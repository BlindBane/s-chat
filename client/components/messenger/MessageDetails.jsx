import React from 'react'
import Header from './../Header'

const MessageDetails = (props) => {
  const { channel, username, messageText, msgId } = props.message
  return (
    <div>
      <Header backToMessenger />
      <p>CHANNEL {channel} </p>
      <p>USER: {username} </p>
      <p>TEXT: {messageText} </p>
      <p>ID: {msgId} </p>
    </div>
  )
}

export default MessageDetails
