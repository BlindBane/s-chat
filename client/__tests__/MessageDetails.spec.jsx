import React from 'react'
import { shallow } from 'enzyme'
import MessageDetails from './../components/messenger/MessageDetails'

import msgs from './../../data.json'

test('MessageDetails renders correctly', () => {
  const msg = msgs.messages[0]
  const component = shallow(<MessageDetails message={msg} />)
  expect(component).toMatchSnapshot()
})
