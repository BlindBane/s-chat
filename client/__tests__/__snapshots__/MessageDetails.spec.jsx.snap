// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageDetails renders correctly 1`] = `
<div>
  <p>
    CHANNEL 
    sanFrancisco
     
  </p>
  <p>
    USER: 
    Bill
     
  </p>
  <p>
    TEXT: 
    Hi, I am bill.
     
  </p>
  <p>
    ID: 
    1
     
  </p>
</div>
`;
